.github/labeler.yml

```diff
@@ -0,0 +1,34 @@
+# https://github.com/actions/labeler
+
+# Add 'repo' label to any root file changes
+"Area: Repo":
+  - "*"
+
+"Area: Components":
+  - packages/paste-core/components/**/*
```

Comment:
Do we want coverage over layout and primitive components? or core?

Reply:
Oh, maybe. One for each?

- `Area: Components`
- `Area: Primitives`
- `Area: Layout`
- `Area: Core bundle`